### PR TITLE
[hotfix][engine][core] Fix execution parallelism not working

### PR DESCRIPTION
**Pull request by liugddx**
## Purpose of this pull request

when the source parallelism is not setting, the parallelism will be set to 1.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by liugddx** [comment]
#2989

File: seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/SeaTunnelClientConfig.java [hunk 1]
```diff
@@ -23,6 +23,15 @@
 
 public class SeaTunnelClientConfig extends ClientConfig {
 
+    public static final InitActionParallelism<Integer, Integer, Integer> INIT_ACTION_PARALLELISM = (actionParallelism, executionParallelism, upstreamParallelism) -> {
```
Comment (ashulin) [inline seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/config/SeaTunnelClientConfig.java]: Such changes are unreasonable.
You can add a utility method to get the correct parallelism when parsing Actions.

**Review by ashulin — COMMENTED** [review]
The modification of Execution DAG & Test classes  is useless

**Comment by liugddx** [comment]
> The modification of Execution DAG & Test classes is useless

already reverted.

**Comment by ashulin** [comment]
@EricJoy2048 @Hisoka-X CC

**Review by ashulin — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**